server/server.js

```diff
@@ -5,12 +5,47 @@ const path = require('path');
 const db = require('../database/database.js');
 
 app.use(express.static('public'));
+app.use(express.json());
 
 app.listen(port, () => {
   console.log(`Listening at http://localhost:${port}`)
-})
+});
 
 app.get('/populate', (req, res) => {
```

Comment:
Did you want this to be an API call? This is almost a post or put request and could allow for other people to change your database. Maybe try populating your database with an npm script

Reply:
...That's fair. Yeah, I wasn't intending that to be for real-life production purposes, of course. It was kind of convenient for me to be able to populate the database via the web browser, but you're right than an npm script makes more sense.